src/main/java/com/hibernate/cinema/model/User.java

```diff
@@ -16,6 +19,8 @@
     @Column(unique = true)
     private String email;
     private String password;
+    @ManyToMany(cascade = CascadeType.ALL)
```

Comment:
What do you think about CascadeType.ALL?

Reply:
Agree it is bad, it was a temporary decision because of an error. Forgot to delete it 👍